language/move-lang/src/parser/ast.rs

```diff
@@ -531,7 +582,19 @@ impl BinOp_ {
             | B::Lt
             | B::Gt
             | B::Le
-            | B::Ge => true,
+            | B::Ge
+            | B::Range
+            | B::Implies => true,
+        }
+    }
+}
+
+impl FunctionVisibility {
+    pub fn is_spec_fun(&self) -> bool {
+        if let FunctionVisibility::Spec(_) = self {
```

Comment:
Nit, I'd pattern match instead of using if let with an else

Also, I'd like to see all the cases stamped out here, no _ pattern

Reply:
Code is now obsolete